[2024-05-06T12:11:32.079+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-06T12:11:32.113+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_operator_dag.insert_to_postgres manual__2024-05-06T12:11:19.872430+00:00 [queued]>
[2024-05-06T12:11:32.125+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_operator_dag.insert_to_postgres manual__2024-05-06T12:11:19.872430+00:00 [queued]>
[2024-05-06T12:11:32.126+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 3
[2024-05-06T12:11:32.144+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): insert_to_postgres> on 2024-05-06 12:11:19.872430+00:00
[2024-05-06T12:11:32.150+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2177) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-06T12:11:32.152+0000] {standard_task_runner.py:63} INFO - Started process 2178 to run task
[2024-05-06T12:11:32.152+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag', 'insert_to_postgres', 'manual__2024-05-06T12:11:19.872430+00:00', '--job-id', '216', '--raw', '--subdir', 'DAGS_FOLDER/kachevo.py', '--cfg-path', '/tmp/tmpw_k4hmvu']
[2024-05-06T12:11:32.155+0000] {standard_task_runner.py:91} INFO - Job 216: Subtask insert_to_postgres
[2024-05-06T12:11:32.215+0000] {task_command.py:426} INFO - Running <TaskInstance: postgres_operator_dag.insert_to_postgres manual__2024-05-06T12:11:19.872430+00:00 [running]> on host db2363de7199
[2024-05-06T12:11:32.376+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='postgres_operator_dag' AIRFLOW_CTX_TASK_ID='insert_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T12:11:19.872430+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T12:11:19.872430+00:00'
[2024-05-06T12:11:32.379+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-06T12:11:32.450+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-05-06T12:11:32.511+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-06T12:11:32.513+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-06T12:11:32.524+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag, task_id=insert_to_postgres, execution_date=20240506T121119, start_date=20240506T121132, end_date=20240506T121132
[2024-05-06T12:11:32.569+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-06T12:11:32.594+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-06T12:11:32.599+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
